src/backend/api/Fusion.Resources.Api/Controllers/Personnel/InternalPersonnelController.cs

```diff
@@ -122,18 +125,19 @@ public InternalPersonnelController()
         [MapToApiVersion("2.0")]
```

Comment:
Could we not just do

```
[MapToApiVersion("1.0")]
[MapToApiVersion("2.0")]
public async Task<ActionResult<ApiCollection<ApiInternalPersonnelPerson>>> GetDepartmentPersonnel(string fullDepartmentString,
            [FromQuery] ODataQueryParams query,
Expand All
	@@ -46,7 +49,7 @@ public InternalPersonnelController()
            [FromQuery] DateTime? timelineEnd = null,
            [FromQuery] bool includeSubdepartments = false,
            [FromQuery] bool includeCurrentAllocations = false)
{
    var command = new GetDepartmentPersonnel(fullDepartmentString, query)
                .IncludeSubdepartments(includeSubdepartments)
                .IncludeCurrentAllocations(includeCurrentAllocations)
                .WithTimeline(shouldExpandTimeline, timelineStart, timelineEnd)
                .WithVersion(2); // No longer use v1
}
```

Could also clean up by just remove v1 logic in the query, so it is not mistaken agan....

Reply:
i tough so to,  but there is something wierd happening making functions dissapear when i do that

Reply:
when combining them the endpoint looks like this:  this should be on top departments/{fullDepartmentString}/resources/personnel
![image](https://github.com/equinor/fusion-app-resources/assets/3900367/6a9311a5-4e8f-4ab7-adbb-6cb783878cd0)

Reply:
Yeh, i know, but i think we can update the swagger config to find it.. They can be reached, they just don't show up in swagger

Reply:
Have updated the swagger logic to handle these endpoints now

Reply:
Now it looks fine, appears to be working as expected.